# Intel compiler have problems with precompiled headers (cannot mix C and C++ precompiled headers, like in GNU compilers)
# B. Hajgato (VUB) April 8th 2016
--- wxPython-src-3.0.2.0/build/tools/build-wxwidgets.py.org	2014-11-27 21:49:59.000000000 +0100
+++ wxPython-src-3.0.2.0/build/tools/build-wxwidgets.py	2016-04-08 13:03:01.362591062 +0200
@@ -268,6 +268,7 @@
             configure_opts.append("--with-gtk=3")
 
         wxpy_configure_opts = [
+                            "--disable-precomp-headers",
                             "--with-opengl",
                             "--enable-sound",
                             "--enable-graphics_ctx",
